### PR TITLE
[No ticket] Minor Accessability fixes

### PR DESCRIPTION
**Pull request by benolot**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: templates/tag/index.html.twig [hunk 1]
```diff
@@ -20,7 +20,7 @@
                 {%- for tag in tagResult.getDomainModels() -%}
                     <li class="br-keyline cf">
                         <a data-istats-link-location="blogs_global_tag" href="{{ path('tag_show', {blogId: blog.getId(), tagId: tag.getId()}) }}">
-                            <h2 class="delta pull--left no-margin">
+                            <h2 class="delta pull--left no-margin" tabindex=0>
```
Comment (Bobkidbob) [inline templates/tag/index.html.twig]: This is not an HTML issue. The `<a>` tag has 0 height and 0 width, causing Safari to skip the link. I will post another comment once I've worked out a solution.

Comment (Bobkidbob) [reply templates/tag/index.html.twig]: This is a float issue. I would recommend using flex to solve the issue by adding the following to the `<a>` tag:
```
display: flex;
justify-content: space-between;
align-items: center;
```
Don't forget to remove `pull--left` and `pull--right` from the children.

There's likely also a clearfix solution, but it's probably better to use flex.

Comment (benolot) [reply templates/tag/index.html.twig]: Outdated review comments

**Pull request merged**